app/src/main/java/com/rahobbs/todo/helpers/TodoItem.java

```diff
@@ -0,0 +1,88 @@
+package com.rahobbs.todo.helpers;
+
+import java.util.Date;
+import java.util.UUID;
+
+/**
+ * Created by rachael on 2/1/16.
+ * Model for a single to-do item
+ */
+public class TodoItem {
+    private String mTitle;
+    private String mDetails;
+    private Date mDate;
+    private UUID mID;
+    private boolean completed;
+    private String mParents;
+    private String mChildren;
+
+
+    public TodoItem() {
+        mID = UUID.randomUUID();
+        mDate = new Date();
+    }
+
+    public TodoItem(UUID id) {
+        mID = id;
+        mDate = new Date();
+    }
+
+    public boolean isCompleted() {
```

Comment:
Also I would separate the getters and the setters. It doesn't matter from a code standpoint, but it makes it easier to read. At a quick glance you can see what is mutable and what isn't